packages/react-composites/src/composites/CallComposite/components/CallPane.tsx

```diff
@@ -0,0 +1,195 @@
+import { ModalLocalAndRemotePIP, ModalLocalAndRemotePIPStyles } from '../../common/ModalLocalAndRemotePIP';
+import { PeoplePaneContent } from '../../common/PeoplePaneContent';
+import { drawerContainerStyles } from '../styles/CallComposite.styles';
+import { TabHeader } from '../../common/TabHeader';
+import { _ICoordinates } from '@internal/react-components';
+import { _pxToRem } from '@internal/acs-ui-common';
+import { useLocale } from '../../localization';
+
+/**
+ * Pane that is used to store participants for Call composite
+ * @private
+ */
+/** @beta */
+export const CallPane = (props: {
+  callAdapter: CallAdapter;
+  onClose: () => void;
+  onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
+  onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
+  onPeopleButtonClicked?: () => void;
+  modalLayerHostId: string;
+  activePane: CallPaneOption;
+  mobileView?: boolean;
+  inviteLink?: string;
+  rtl?: boolean;
+}): JSX.Element => {
+  const [drawerMenuItems, setDrawerMenuItems] = useState<_DrawerMenuItemProps[]>([]);
+
+  const hidden = props.activePane === 'none';
+  const paneStyles = hidden ? hiddenStyles : props.mobileView ? availableSpaceStyles : sidePaneStyles;
+  const localeStrings = useLocale();
+
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+  const useStrings = () => {
+    /* @conditional-compile-remove(call-composite-participant-pane) */
+    return localeStrings.strings.call;
+
+    return localeStrings.strings.callWithChat;
+  };
+
+  const strings = useStrings();
+  const theme = useTheme();
+
+  const header =
+    props.activePane === 'none' ? null : props.mobileView ? (
+      <TabHeader {...props} strings={strings} activeTab={props.activePane} />
+    ) : (
+      <SidePaneHeader
+        {...props}
+        strings={strings}
+        headingText={props.activePane === 'people' ? strings.peoplePaneTitle : ''}
+      />
+    );
+
+  /**
+   * In a Call Composite when a participant is removed, we must remove them from the call.
+   */
+  const removeParticipantFromCall = async (participantId: string): Promise<void> => {
+    await props.callAdapter.removeParticipant(participantId);
+  };
+
+  const peopleContent = (
+    <CallAdapterProvider adapter={props.callAdapter}>
+      <PeoplePaneContent
+        {...props}
+        onRemoveParticipant={removeParticipantFromCall}
+        setDrawerMenuItems={setDrawerMenuItems}
+        strings={strings}
+      />
+    </CallAdapterProvider>
+  );
+
+  // Use document.getElementById until Fluent's Stack supports componentRef property: https://github.com/microsoft/fluentui/issues/20410
+  const modalLayerHostElement = document.getElementById(props.modalLayerHostId);
+  const modalHostRef = useRef<HTMLElement>(modalLayerHostElement);
+  const modalHostWidth = _useContainerWidth(modalHostRef);
+  const modalHostHeight = _useContainerHeight(modalHostRef);
+  const minDragPosition: _ICoordinates | undefined = useMemo(
```

Comment:
Possible to move these util functions to `common/utils.tsx` and re-use them in both call and callwithchat?
I feel uneasy about duplicating such complex logic

Reply:
Good point, I moved it out into a util file so that it can be used in both CallPane and CallWithChatPane.
`common/utils.tsx`

Reply:
Functioning correctly in CallWithChat after moving to `common/utils.tsx`
https://user-images.githubusercontent.com/97130533/180077645-17778263-9bb6-4e1d-b3dc-2448e007e68a.mov